python/pyproject.toml

```diff
@@ -16,7 +16,7 @@ whylogs-sketching = ">=3.4.1.dev3"
 protobuf = ">=3.19.4"
 importlib-metadata = { version = "<4.3", python = "<3.8" }
 typing-extensions = {version = ">=3.10", markers = "python_version < \"4\""}
-whylabs-client = "^0.5.10"
+whylabs-client = "0.6.0"
```

Comment:
we could use the carrot there so we don't need to update whylogs on each patch release.

Reply:
added back :)